apps/cookbook/src/app/showcase/chart-showcase/chart-showcase.component.html

```diff
@@ -1,7 +1,8 @@
 <div class="example">
   <cookbook-chart-example></cookbook-chart-example>
   <div class="code-example">
```

Comment:
Use `cookbook-example-viewer` and lign with existing examples/showcase:
```suggestion
  <cookbook-example-viewer [html]="monthlyOverviewExample.template" [ts]="monthlyOverviewExample.codeSnippet">
    <cookbook-chart-example-monthly-overview #monthlyOverviewExample></cookbook-chart-example-monthly-overview>
  </cookbook-example-viewer>
```